packages/common/src/components/Preview/index.tsx

```diff
@@ -363,6 +363,10 @@ class BasePreview extends React.PureComponent<Props, State> {
             this.setState({ showScreenshot: false });
             break;
           }
+          case 'document-focus': {
+            trackWithCooldown('Browser focus', 5000);
```

Comment:
@danilowoz I think it would be enough to send an event every 30 sec. Also, I'd name it as `Preview focus`.
The rest looks good to me!